devices/schneider_electric.js

```diff
@@ -384,13 +417,32 @@ module.exports = [
         model: '41EPBDWCLMZ/354PBDMBTZ',
         vendor: 'Schneider Electric',
         description: 'Wiser 40/300-Series Module Dimmer',
-        fromZigbee: [fz.on_off, fz.brightness, fz.level_config, fz.lighting_ballast_configuration, fzLocal.indicator_mode],
-        toZigbee: [tz.light_onoff_brightness, tz.level_config, tz.ballast_config, tzLocal.indicator_mode],
+        fromZigbee: [
+            fz.on_off,
+            fz.brightness,
+            fz.level_config,
+            fz.lighting_ballast_configuration,
+            fzLocal.indicator_mode,
+            fzLocal.dimmer_restore_level,
+        ],
+        toZigbee: [
+            tz.light_onoff_brightness,
+            tz.level_config,
+            tz.ballast_config,
+            tzLocal.indicator_mode,
+            tzLocal.dimmer_restore_level,
```

Comment:
Why is this converter needed, `tz.level_config` already support `on_level`: https://github.com/Koenkk/zigbee-herdsman-converters/blob/005926edd29de88d9bc1ee18b05c102a5caff145/converters/toZigbee.js#L567

Reply:
Thanks - I didn't do my research. 
I will incorporate the base functions.